content/en/docs/setup/production-environment/windows/user-guide-windows-containers.md

```diff
@@ -26,7 +26,7 @@ This guide walks you through the steps to configure and deploy a Windows contain
 ## Before you begin
 
 * Create a Kubernetes cluster that includes a 
-[master and a worker node running Windows Server](/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes)
+master and a [worker node running Windows Server](/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes)
```

Comment:
Hi @jihoon-seo .
Can you replace the word "master" with "control plane" node?

Reply:
Sure, I will do that soon!